crates/ln-dlc-node/src/ln/event_handler.rs

```diff
@@ -184,6 +184,9 @@ where
                     PaymentPurpose::SpontaneousPayment(preimage) => (Some(preimage), None),
                 };
 
+                // TODO: Check if there is a corrsponding ff payment in the databse and update it to
+                // paid if so TODO: Increase fee counter to track fee revenue
```

Comment:
🤔 You won't be able to do this directly in this unfortunately generic event handler.

You can do something like what @holzeis did in the app to subscribe to these events outside of the event handler.

Reply:
Sorry, this TODO should not be part of the changeset but will be done on top! This should have not sneaked in :)

Reply:
Oh yeah, that's okay. I'm just letting you know in case it wasn't already clear.